frontend/src/components/OmakaseModal.jsx

```diff
@@ -0,0 +1,209 @@
+                  setFormData((prevFormData) => ({
+                    ...prevFormData,
+                    format: detail.selectedOption.value,
+                  }))
+                }
+                options={formats.map((format) => ({
+                  label: format,
+                  value: format,
+                }))}
+              />
+            </FormField>
+          )}
+
+          {formData.operation === "Flow Creation" && (
+            <FormField label="Label">
+              <Input
+                value={formData.label}
+                onChange={({ detail }) =>
+                  setFormData({ ...formData, label: detail.value })
+                }
+                placeholder="Label"
+              />
+            </FormField>
+          )}
+
+          <SpaceBetween direction="vertical" size="s">
+            {flows
+              .filter((flow) => flow.format === "urn:x-nmos:format:audio")
+              .map((flow) => (
+                <Checkbox
+                  key={flow.id}
+                  checked={formData.flows[flow.id] || false}
+                  onChange={({ detail }) =>
+                    setFormData((prev) => ({
+                      ...prev,
+                      flows: { ...prev.flows, [flow.id]: detail.checked },
+                    }))
+                  }
+                >
+                  {flow.description ?? ""}
+                </Checkbox>
+              ))}
+          </SpaceBetween>
+
+          <ExpandableSection
+            headerText="Advanced"
+            variant="footer"
+            expanded={showAdvancedContent}
+            onChange={({ detail }) => setShowAdvancedContent(detail.expanded)}
+          >
+            <SpaceBetween direction="vertical" size="m">
+              <FormField label="Bucket">
+                <Input
+                  value={formData.bucket}
+                  onChange={({ detail }) =>
+                    setFormData({ ...formData, bucket: detail.value })
+                  }
+                  placeholder="Bucket"
+                />
+              </FormField>
+              <FormField label="Path">
+                <Input
+                  value={formData.path}
+                  onChange={({ detail }) =>
+                    setFormData({ ...formData, path: detail.value })
+                  }
+                  placeholder="Path"
+                />
+              </FormField>
+              <FormField label="Filename">
+                <Input
+                  value={formData.filename}
+                  onChange={({ detail }) =>
+                    setFormData({ ...formData, filename: detail.value })
+                  }
+                  placeholder="Filename"
+                />
+              </FormField>
+            </SpaceBetween>
+          </ExpandableSection>
+
+          <Box float="right">
+            <SpaceBetween direction="horizontal" size="xs">
+              {isSpinnerVisible && <Spinner />}
+              <Button
```

Comment:
Should be able to use the `loading` prop of the button to avoid the need of the spinner?

Reply:
good point